### ISSUE TITLE
[Accessibility] A11y_.Net Editor_Keyboard: Focus is not visible on the “Enter focus mode” and “Run” button

### ISSUE THREAD

**Issue by Tanushagottipati** [dotnet/try#106]
**Environment Details**
https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User 
**Browser:** Chrome (67.0.3396.87)

**Additional Details:**
MAS Violated: MAS2.4.7

**Repro steps:**
1. Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User 
2. Tab till “Enter focus mode” and press enter
3. Tab till “Run” button.
4. Verify Whether the focus is visible on the “Enter focus mode” and “Run” button.

**Actual Result:**
Focus is not visible on the “Enter focus mode” and “Run” button. 

**Expected Result:**
Focus must be properly visible for the users in order to access the buttons.

**User Impact:**
Keyboard users find it difficult to understand where the focus is if the focus is not visible clearly.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}
![keyboard focus is not visible on enter focus mode](https://user-images.githubusercontent.com/41109135/42629743-1788d492-85f2-11e8-9d54-3897617896b8.png)
![keyboard focus is not visible on the run](https://user-images.githubusercontent.com/41109135/42629745-17bdedf8-85f2-11e8-9e8e-00665c9aa57a.png)

**Comment by LadyNaggaga** [comment]
Duplicate entry https://github.com/dotnet/try/issues/104

**Comment by umasarath52** [comment]
This was discussed with Ryan and approved to close as this is non blocking scenario.
